app/models/corporate_information_page.rb

```diff
@@ -175,13 +175,15 @@ def alternative_format_provider_required?
   end
 
   def base_path
-    if worldwide_organisation.present?
-      url = edition_worldwide_organisation.worldwide_organisation.base_path.to_s + "/about/#{slug}"
-      url.gsub("/about/about", "")
-    elsif organisation.present?
-      url = edition_organisation.organisation.base_path.to_s + "/about/#{slug}"
-      url.gsub("/about/about", "/about")
-    end
+    return if owning_organisation.blank?
+
+    url = owning_organisation.base_path + "/about/#{slug}"
+    # The corporate information pages associated with worldwide organisations are not rendered
+    # on their own page. Instead, we publish them as `placeholder` content items with the path
+    # /about/about`. On the other hand, the corporate information pages associated with organisations
+    # are rendered on their own page, which we want to have the path `/about`.
+    url.gsub!("/about/about", "/about") if organisation.present?
```

Comment:
I think adding an explanation of this line to the commit message would suffice to remove comments. Basically just saying that the presence of an organisation rather than a worldwide_organisation is enough for us to work out what's going on